docs/topics/azure-stack.md

```diff
@@ -145,6 +146,126 @@ Each AKS Engine release is validated and tied to a specific version of the AKS B
 
 Azure Monitor for containers can be deployed to AKS Engine clusters hosted in Azure Stack Hub Cloud Environments. Refer to [Azure Monitor for containers](../topics/monitoring.md#azure-monitor-for-containers) for more details on how to onboard and monitor clusters, nodes, pods, containers inventory, performance metrics and logs.
 
+## Volume Provisioner: CSI Drivers (preview)
+As a [replacement of the current in-tree volume provisioner](https://kubernetes.io/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/), three CSI Drivers are avaiable on Azure Stack Hub. Please find details in the following table.
```

Comment:
`As a replacement to the deprecated in-tree...` may provide more context to the reader.

Maybe including the term `storage` somewhere in the title will help novice users to understand faster what is this about.